### PR TITLE
Flatten children with React.Children.toArray upfront

### LINKED ISSUE #62: Not loading all the steps
I've having issue where some of the steps are not loaded even thought they are inside the StepWizard component. Let's say I have 8 steps, it only shows the first 5. What could be the problem here?

This doesn't work:
```
{
   questionnaires.map((questionnaire) => (
     <Questionnaire question={questionnaire.title} options={questionnaire.options} callback={updateSymptom} />
    ))
}

```

This works:
```
<Questionnaire question={questionnaires[0].title} options={questionnaires[0].options} callback={updateSymptom} />
<Questionnaire question={questionnaires[1].title} options={questionnaires[1].options} callback={updateSymptom} />
<Questionnaire question={questionnaires[2].title} options={questionnaires[2].options} callback={updateSymptom} />
<Questionnaire question={questionnaires[3].title} options={questionnaires[3].options} callback={updateSymptom} />
```

### LINKED ISSUE #65: Failure if there is only one child of the stepwizard
If you have only one child component in the step wizard, you get this error
×
TypeError: o.props.children.filter is not a function
a.getSteps
node_modules/react-step-wizard/dist/react-step-wizard.min.js:1

### PR DESCRIPTION
**Pull request by resolritter**
https://reactjs.org/docs/react-api.html#reactchildrentoarray

closes #65: in the single child case, where React will turn it into [child]
closes #62: on nested JSX expressions which map over a data structure, React.Children.toArray will flatten everything into a single list of children

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ export default class StepWizard extends PureComponent {
         return this.getSteps().length;
     }
 
-    getSteps = () => this.props.children.filter(el => el);
+    getSteps = () => React.Children.toArray(this.props.children).filter(el => el);
```
Comment (resolritter) [inline src/index.js:139]: I'm not sure if this `filter(el => el)` is still needed. To my current understanding, it's useful for removing `null` nodes, but `React.Children.toArray` might already do this.

Comment (jcmcneal) [reply src/index.js:139]: I'd prefer if we didn't have to do a 2nd iteration to remove null nodes. Can you write a unit test with some null children to validate if we still need the `filter`?

Comment (resolritter) [reply src/index.js]: From my findings, it's not needed anymore. See 06debaa.

Comment (jcmcneal) [reply src/index.js]: Great! Let’s merge!

**Comment by resolritter** [comment]
@jcmcneal to be clear, do you want me to merge it myself or will you do it?

**Review by jcmcneal — APPROVED** [review]

**Pull request merged**